core/unix/signal_linux_x86.c

```diff
@@ -210,14 +210,37 @@ save_xmm(dcontext_t *dcontext, sigframe_rt_t *frame)
 #ifdef X64
         /* Some assemblers, including on Travis, don't know "xsave64", so we
          * have to use raw bytes for:
-         *    48 0f ae 20  xsave64 (%rax)
+         * 48 0f ae 21  xsave64 (%rcx)
```

Comment:
nit: Easier to read when indented as it was.

Reply:
Done.